src/bundle/Resources/public/js/scripts/helpers/notification.helper.js

```diff
@@ -10,7 +10,10 @@
      * Dispatches notification event
      *
      * @function showNotification
-     * @param {{message: string, label: string}} detail
+     * @param {Object} detail
+     * @param {String} detail.message
+     * @param {String} detail.label
+     * @param {Function} [detail.callback] Called after notification Node was added
      */
     const showNotification = (detail) => {
```

Comment:
Ok, you're passing the callback here, but when it's invoked?

Reply:
Hmm, `Called after notification Node was added` from jsDoc is not precise enough?
Callback is called right after we append notification element to the DOM.

Reply:
The thing is you're dispatching it with an event. So it may be invoked and may be not as well. What's more, I'm missing an information about the kind of this callback. Is it invoked on success or on fail?

Reply:
>  Is it invoked on success or on fail?
I don't know what do you mean by _fail_?
Could you please specify what how you want it to look like? You want different name for callback, different description in jsDoc, different behaviour of the whole mechanism?

Reply:
1. Please, change the description of `detail.callback` because it promises the callback will be invoked (but it depends on the final implementation of event listener),
2. The `callback` is very generic name. You don't know what is it for. I'm missing context here.

Reply:
1. I will change description to: `to be called after notification Node was added`, which does not tell that it will be invoked but rather that it is intended to be invoked on notification show.
2. I think we may change name to `onShow`. I googled a bit and found that theres even something similar for notifications in PWA: https://developer.mozilla.org/en-US/docs/Web/API/Notification/onshow

Reply:
I like it. Go on!